source/gui/exit.py

```diff
@@ -43,10 +44,16 @@ def _displayStringLabels(self):
 			self.EXIT: _("Exit"),
 			# Translators: An option in the combo box to choose exit action.
 			self.RESTART: _("Restart"),
+			self.RESTART_WITH_ADDONS_DISABLED_AND_DEBUG_LOGGING: _(
+				# Translators: An option in the combo box to choose exit action.
+				"Restart with add-ons disabled and debug logging",
+			),
+			self.RESTART_WITH_ADDONS_DISABLED: _(
+				# Translators: An option in the combo box to choose exit action.
+				"Restart with add-ons disabled",
+			),
 			# Translators: An option in the combo box to choose exit action.
-			self.RESTART_WITH_ADDONS_DISABLED: _("Restart with add-ons disabled"),
-			# Translators: An option in the combo box to choose exit action.
-			self.RESTART_WITH_DEBUG_LOGGING_ENABLED: _("Restart with debug logging enabled"),
+			self.RESTART_WITH_DEBUG_LOGGING: _("Restart with debug logging"),
```

Comment:
I don't think these wording changes actually improve things for users, and may in fact make things less clear.

You said:
> technically there is no debug logging that we can enable or disable in NVDA, but a debug level that may be set to "disabled", to "debug" or other intermediate levels.

That's as may be, but effectively what you're doing when you set a debug level is enabling all debug levels which are less than or equal to that debug level, so we are, in fact, enabling debug logging.

Reply:
OK, I trust you for what sounds better in this case and I understand your reasoning.

Though the first reason why I have changed this in the first place is to be similar with "Restart with add-ons disabled and debug logging" item. So I should change it anyway to "Restart with add-ons disabled and debug logging enabled". But I was concerned that this wording was a bit long in the combo-box. What do you think?

If you do not share the same concerns, I will add "enabled" for both items containing debug logging.

Reply:
Hmm, I definitely see what you mean about it being quite long. Probably @seanbudd or a sighted dev would be best to answer that, I tend to only use magnification as a last resort.

Reply:
Re the visual aspect, the text is not cropped. The length of the window correctly adapts to its content.

But actually, I have written "quite long" thinking to the combo-box item to be heard (i.e. long for an choice label), not to the visual aspect.

Reply:
@CyrilleB79, in conversation with @michaelDCurran, I think "debug logging enabled" is the most understandable wording. Ideally we would rework the exit dialog though, but as you say, that has been stalled for a long time, and the perfect is the enemy of the good.

Reply:
OK, I have restored the original wording "Restart with debug logging enabled" and updated the new wording similarly.